### ISSUE TITLE
Yakuza 3 - Fail at importing big stages

### ISSUE THREAD

**Issue by MrZasen** [theturboturnip/yk_gmd_io#28]
So I tried to import this stage, resulting in this failure:
```
Importing st_tojo_n_outside.gmd...
Extracting abstract scene...
Traceback (most recent call last):
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\blender\importer\gmd_importers.py", line 260, in execute
    gmd_scene = read_abstract_scene_from_filedata_object(gmd_version, False, gmd_contents, error)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\io.py", line 88, in read_abstract_scene_from_filedata_object
    return GMDAbstractor_YK1(version_props, can_have_skinned_vertices, cast(FileData_YK1, contents), error_reporter).make_abstract_scene()
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\yk1\to_abstract.py", line 48, in make_abstract_scene
    bytestrings_are_16bit)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\common\to_abstract.py", line 324, in build_meshes_from_structs
    triangle_indices, min_index, max_index = process_indices(mesh_struct, mesh_struct.triangle_list_indices)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\common\to_abstract.py", line 319, in process_indices
    indices.append(index)
OverflowError: unsigned short is less than minimum

location: <unknown location>:-1
```

**Comment by MrZasen** [comment]
```
Traceback (most recent call last):
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\blender\importer\gmd_importers.py", line 260, in execute
    gmd_scene = read_abstract_scene_from_filedata_object(gmd_version, False, gmd_contents, error)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\io.py", line 88, in read_abstract_scene_from_filedata_object
    return GMDAbstractor_YK1(version_props, can_have_skinned_vertices, cast(FileData_YK1, contents), error_reporter).make_abstract_scene()
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\yk1\to_abstract.py", line 48, in make_abstract_scene
    bytestrings_are_16bit)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\common\to_abstract.py", line 324, in build_meshes_from_structs
    triangle_indices, min_index, max_index = process_indices(mesh_struct, mesh_struct.triangle_list_indices)
  File "C:\Users\x\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\yk_gmd_blender\yk_gmd\v2\converters\common\to_abstract.py", line 319, in process_indices
    indices.append(index)
OverflowError: unsigned short is less than minimum

location: <unknown location>:-1
```

**Comment by MrZasen** [comment]
So it seems it isn't related to big stages, since the script has no problem importing some 3.1Mb gmd, and then fails when trying with a 3.0Mb file.

**Comment by theturboturnip** [comment]
Will be fixed in upcoming v0.4.0, see PR #48
